server/handlers/handlers.go

```diff
@@ -101,6 +102,36 @@ func (ah *ActivityHandler) Start() {
 		}
 	}
 
+	// doStart is the meat of the activity handler worker
+	doStartLastActivityAt := func() {
+		for {
+			timer := time.NewTimer(5 * time.Minute)
+			select {
+			case <-timer.C:
+				ah.lastUpdateAtMap.Range(func(subscriptionID, lastUpdateAt any) bool {
+					if lastUpdateAt.(time.Time).Add(5 * time.Minute).After(time.Now()) {
```

Comment:
nit: I had to sit down and think about what this meant. Maybe if we expressed it as the following instead?
```go
if time.Now().Sub(lastUpdateAt) <= 5 * time.Minute
```